uploadgui/src/main/scala/se/lu/nateko/cp/meta/upload/FileHasher.scala

```diff
@@ -15,13 +15,24 @@ import se.lu.nateko.cp.meta.core.crypto.Sha256Sum
 
 object FileHasher {
 
+	val FileMaxSize = 2_000_000_000
+	val FileMaxSizeMessage = "This file could be too large to upload with this web app. " +
+		"Please consider scripted uploads as instructed here: " +
+		"https://github.com/ICOS-Carbon-Portal/meta/?tab=readme-ov-file#upload-instructions-scripting"
+
```

Comment:
I would probably write:
> The file you selected is too large to upload with this form. Please use scripted uploads as described in [our documentation](https://github.com/ICOS-Carbon-Portal/meta/?tab=readme-ov-file#upload-instructions-scripting).

Reply:
It would be nice to use an HTML link, but you have to see if it's possible to pass it through the exception.